Filezilla stopped working

## Description
I had Filezilla working previously (it was on the beta and development channels at the time). Now, I'm on the stable channel (where I should be with Chromebrew) and Filezilla doesn't load. 

If you executed a specific command:
```shell
filezilla
```

Output:
```shell
Reading locale option from /home/chronos/user/.config/filezilla/filezilla.xml
GLib-GIO-Message: 20:51:09.338: Using the 'memory' GSettings backend.  Your settings will not be saved or shared with other applications.
Floating point exception (core dumped)
```

## Versions and system information
chromebrew: `1.0.6`

ChromeOS: `CHROMEOS_AUSERVER=https://tools.google.com/service/update2
CHROMEOS_BOARD_APPID={BABC2E82-8F7D-26E5-EE66-4C60A2B41DBC}
CHROMEOS_CANARY_APPID={90F229CE-83E2-4FAF-8479-E368A34938B1}
CHROMEOS_DEVSERVER=
CHROMEOS_RELEASE_APPID={BABC2E82-8F7D-26E5-EE66-4C60A2B41DBC}
CHROMEOS_RELEASE_BOARD=candy-signed-mp-v2keys
CHROMEOS_RELEASE_BRANCH_NUMBER=59
CHROMEOS_RELEASE_BUILDER_PATH=candy-release/R71-11151.59.0
CHROMEOS_RELEASE_BUILD_NUMBER=11151
CHROMEOS_RELEASE_BUILD_TYPE=Official Build
CHROMEOS_RELEASE_CHROME_MILESTONE=71
CHROMEOS_RELEASE_DESCRIPTION=11151.59.0 (Official Build) stable-channel candy 
CHROMEOS_RELEASE_KEYSET=mp-v2
CHROMEOS_RELEASE_NAME=Chrome OS
CHROMEOS_RELEASE_PATCH_NUMBER=0
CHROMEOS_RELEASE_TRACK=stable-channel
CHROMEOS_RELEASE_VERSION=11151.59.0
DEVICETYPE=CHROMEBOOK
GOOGLE_RELEASE=11151.59.0`

Model: Dell Chromebook 11 3120

Architecture: `x86_64`

So I did a powerwash because I was having problems with SFTP on Sublime Text, and I wanted to give Filezilla another shot.

I re-installed Chromebrew, and the first thing I tried to install was Filezilla. But, it looks like I can't. I get to this...

patch: Patch takes a patch file containing a difference listing produced by the diff program and applies those differences to one or more original files, producing patched versions.
http://savannah.gnu.org/projects/patch/
version 2.7.6
Downloading source...
################################################################################################################################################ 100.0%
Archive downloaded
Unpacking archive, this may take awhile...
gcc (GCC) 7.3.0
Copyright (C) 2017 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.

Building from source, this may take a while...
Rename all *.la files to *.la_tmp
checking for a BSD-compatible install... /usr/bin/install -c
checking whether build environment is sane... yes
checking for a thread-safe mkdir -p... /bin/mkdir -p
checking for gawk... gawk
checking whether make sets $(MAKE)... no
checking whether make supports nested variables... no
checking whether make supports nested variables... (cached) no
checking for gcc... gcc
checking whether the C compiler works... no
configure: error: in `/usr/local/tmp/crew/patch-2.7.6.tar.xz.dir/patch-2.7.6':
configure: error: C compiler cannot create executables
See `config.log' for more details
patch failed to install: `./configure --prefix=/usr/local --libdir=/usr/local/lib64` exited with 77

--- a/packages/filezilla.rb
+++ b/packages/filezilla.rb
@@ -3,21 +3,21 @@ require 'package'
 class Filezilla < Package
   description 'FileZilla Client is a free FTP solution.'
   homepage 'https://filezilla-project.org/'
-  version '3.39.0'
-  source_url 'https://download.filezilla-project.org/client/FileZilla_3.39.0_src.tar.bz2'
-  source_sha256 'fb515e93775e5ad94ff2755b39605b62b3e8fcfc1125757f411d8f580d16444f'
+  version '3.43.0'
+  source_url 'https://download.filezilla-project.org/client/FileZilla_3.43.0_src.tar.bz2'
+  source_sha256 '6fe652c360682f860abe2dc2e196bddca6260242cefe6fe779d42c327c01258e'
 
   binary_url ({
-    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/filezilla-3.39.0-chromeos-armv7l.tar.xz',
-     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/filezilla-3.39.0-chromeos-armv7l.tar.xz',
-       i686: 'https://dl.bintray.com/chromebrew/chromebrew/filezilla-3.39.0-chromeos-i686.tar.xz',
-     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/filezilla-3.39.0-chromeos-x86_64.tar.xz',
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/filezilla-3.43.0-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/filezilla-3.43.0-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/filezilla-3.43.0-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/filezilla-3.43.0-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
-    aarch64: '7da4e0d6fc406bfd0e321ebd9058a2346ac50c182c62b201355be1c65f80e3f1',
-     armv7l: '7da4e0d6fc406bfd0e321ebd9058a2346ac50c182c62b201355be1c65f80e3f1',
-       i686: 'd6a0fd020e660b70bd64f1068c349e5cbe01e99255b9d931ad36d3b5c0937d06',
-     x86_64: 'b3cd0a4569aa2396003683e824381f9856d1353fb79b47564f8887644b0cc170',
+    aarch64: 'fdef2404efb7251ba6c9b3675d6f7364b57272cc8c431e8de69761224bae5f9d',
+     armv7l: 'fdef2404efb7251ba6c9b3675d6f7364b57272cc8c431e8de69761224bae5f9d',
+       i686: '652990bad986fc083e5d9137bf9df6bba7479ca8ec255a9250c34680b5676610',
+     x86_64: '3fe771c921d4738d0c82d33bfd0aeea1405009ce0ea7d126603801f40abec1ee',
   })
 
   depends_on 'dbus'

--- a/packages/libfilezilla.rb
+++ b/packages/libfilezilla.rb
@@ -3,21 +3,21 @@ require 'package'
 class Libfilezilla < Package
   description 'libfilezilla is a small and modern C++ library, offering some basic functionality to build high-performing, platform-independent programs.'
   homepage 'https://lib.filezilla-project.org/'
-  version '0.15.1'
-  source_url 'https://download.filezilla-project.org/libfilezilla/libfilezilla-0.15.1.tar.bz2'
-  source_sha256 '2048c4128f3bf37a2a4ece17c8bea5455f3d7414fe2e060afcf2a8b00a87f49f'
+  version '0.18.0'
+  source_url 'https://download.filezilla-project.org/libfilezilla/libfilezilla-0.18.0.tar.bz2'
+  source_sha256 '577a663b5dcf7f44e76a4208f11d6e7c4325c8f45fe15be8588de86eb75f9f3c'
 
   binary_url ({
-    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/libfilezilla-0.15.1-chromeos-armv7l.tar.xz',
-     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/libfilezilla-0.15.1-chromeos-armv7l.tar.xz',
-       i686: 'https://dl.bintray.com/chromebrew/chromebrew/libfilezilla-0.15.1-chromeos-i686.tar.xz',
-     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/libfilezilla-0.15.1-chromeos-x86_64.tar.xz',
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/libfilezilla-0.18.0-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/libfilezilla-0.18.0-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/libfilezilla-0.18.0-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/libfilezilla-0.18.0-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
-    aarch64: '9f0c812627b00d80513fd6efa9f26151f7fd91be28277f387fe8f950fe59d7ac',
-     armv7l: '9f0c812627b00d80513fd6efa9f26151f7fd91be28277f387fe8f950fe59d7ac',
-       i686: '85e40754a5dcb32e2f77b2bd95e0e25d1a8319e9a8bb09688596b44c279ba715',
-     x86_64: '607e93feabe4f830aa05379bd1a1b9930f6253ba9ab3bede5ee56f60ac542de5',
+    aarch64: 'd35b79ea3d8d2854986c3880bf6b39792914b7c90a8e41c3dc8af1616a897ca9',
+     armv7l: 'd35b79ea3d8d2854986c3880bf6b39792914b7c90a8e41c3dc8af1616a897ca9',
+       i686: '58437bfb3359931446837389c2116d3d4f997069d1a39a901ce6bfc47ef652e5',
+     x86_64: '87dd446e4984dedb2ffb7f704a3458a29c64fd8cba33d49ac7bdd3241d53bc7a',
   })
 
   def self.build

